media/common/src/main/java/io/helidon/media/common/ContentWriters.java

```diff
@@ -147,4 +187,18 @@ private ContentWriters() {
     public static Function<ReadableByteChannel, Publisher<DataChunk>> byteChannelWriter() {
         return byteChannelWriter(null);
     }
+
+    /**
+     * Returns a writer function for a {@link Throwable}'s stack trace.
+     *
+     * @param context a {@link MessageBodyWriterContext}
+     * @param setContentLength whether {@link MessageBodyWriterContext#contentLength(long)} should be called
+     * @return a {@link Throwable} writer
+     * @see #writeStackTrace(Throwable, MessageBodyWriterContext, boolean)
+     */
+    public static Function<Throwable, Publisher<DataChunk>> stackTraceWriter(MessageBodyWriterContext context,
```

Comment:
`Function<T, Publisher<DataChunk>` is the old style of writer that is deprecated, we don't need that.

Reply:
OK; will remove these.  (The other ones: should they be `@Deprecated`?)